app-native/src/main.rs

```diff
@@ -22,28 +33,71 @@ enum MainError {
   StudioWriteLock,
 }
 
+pub struct FakeBusNode<'a> {
+  name: &'a str,
+  class: NodeClass,
+  features: HashSet<NodeFeature>,
+}
+
+impl<'a> BusNode for FakeBusNode<'a> {
+  fn name(&self) -> &str {
+    self.name
+  }
+
+  fn class(&self) -> &NodeClass {
+    &self.class
+  }
+
+  fn features(&self) -> &HashSet<NodeFeature> {
+    &self.features
+  }
+
+  fn send(&mut self, time: ClockTime, msg: &Message) {
+    println!(">>> {:?} {:?}", time, msg)
+  }
+}
+
 fn main() -> Result<(), Error> {
   let config_path =
     std::env::var(HERO_STUDIO_CONFIG).unwrap_or(DEFAULT_HERO_STUDIO_CONFIG.to_string());
```

Comment:
It seems it's always preferred to use `unwrap_or_else` instead of `unwrap_or` as this is eagerly evaluated.